package.json

```diff
@@ -1,7 +1,6 @@
 {
   "license": "UNLICENSED",
   "devDependencies": {
-    "@outreach/semantic-release-circleci-orb": "^1.1.9",
```

Comment:
looks like we added it [manually](https://github.com/getoutreach/devbase/commit/86f9a4e0dd714d427f6144b83caac5c373926056). should it be retained?

Reply:
Yeah, we do need to keep this. Thanks for the catch!

I am going to close this PR, however, and we can re-stencil with the next version.